src/Uno.UI/Microsoft/UI/Xaml/Controls/PipsPager/PipsPagerSelectedIndexChangedEventArgs.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.UI.Xaml.Controls
+{
+	public class PipsPagerSelectedIndexChangedEventArgs
+	{
+		internal PipsPagerSelectedIndexChangedEventArgs(int newPageIndex, int previousPageIndex) =>
+			(NewPageIndex, PreviousPageIndex) = (newPageIndex, previousPageIndex);
+
+		public int NewPageIndex { get; }
+
+		public int PreviousPageIndex { get; }
```

Comment:
![image](https://user-images.githubusercontent.com/31348972/134734213-1cd71adb-4f5e-4dab-b66c-904338daca0b.png)

![image](https://user-images.githubusercontent.com/31348972/134734229-14985db5-a575-4e59-afff-5f12d5ca3d6f.png)

What I'm seeing in WinUI 2.7 looks very weird..

Reply:
This was in the old version, they removed the properties since, now my code is up to date :-) .